src/style.css

```diff
@@ -65,19 +67,87 @@
 }
 
 
+/* hover effect section */
+.home {
+  color: black;
+  transition: 0.3s;
+}
+
+.explore {
+  color: black;
+  transition: 0.3s;
+}
+
+.notifications {
+  color: black;
+  transition: 0.3s;
+}
+
+.messages {
+  color: black;
+  transition: 0.3s;
+}
+
+.bookmarks {
+  color: black;
+  transition: 0.3s;
+}
+
+.profile {
+  color: black;
+  transition: 0.3s;
+}
+
+.more{
+  color: black;
+  transition: 0.3s;
+}
+
+
+
+.home:hover {
```

Comment:
saw your last commit mentioned your hover styles aren't working - I think this is because you have given the classes that you are using as a selector to the img element instead of the nav element. If I understand this correctly, your hover styles are trying to change the color of the img, not the text.  I'd try creating a class to use with :hover {}, and adding it to the elements that need it.

Reply:
omg thank you !! i will give that a shot :)))